CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 ### 🐞 Bug Fixes
 
 * Fixed an issue preventing `FormField` labels from rendering if `fieldDefaults` was undefined.
+* Admin activity tracker now displays username of user being impersonated, instead of true.
```

Comment:
This is the kind of thing we can leave out of changelog, I think -- its a tooling tweak that never really effected anybody's app...